pkgs/development/python-modules/runstats/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  setuptools,
+  cython,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pytest-cov-stub,
+  pytest-xdist,
+}:
+
+buildPythonPackage rec {
+  pname = "runstats";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "grantjenks";
+    repo = "python-runstats";
+    tag = "v${version}";
+    hash = "sha256-YF6S5w/ccWM08nl9inWGbaLKJ8/ivW6c7A9Ny20fldU=";
+  };
+
+  build-system = [
+    setuptools
+    cython
```

Comment:
Is `cpython` here necessary? The package seems to build correctly without it.

Reply:
It is needed for a faster implementation. But it has a fallback vanilla python implementation too. It raises warnings without it. IMO the whole point of this package is in the cpython acceleration.

Reply:
Alright, thanks for the clarification.

Reply:
No no, i thank you spending your time for reviewing.